wp-content/themes/pub/wporg-learn-2024/src/learning-pathway-header/index.php

```diff
@@ -0,0 +1,118 @@
+			<!-- wp:group {"style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60"}},"layout":{"selfStretch":"fill","flexSize":null}},"layout":{"type":"constrained"}} -->
+			<div class="wp-block-group" style="padding-top:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60)">
+
+				<!-- wp:heading {"level":1,"fontSize":"heading-1","fontFamily":"eb-garamond"} -->
+				<h1 class="wp-block-heading has-eb-garamond-font-family has-heading-1-font-size">' . esc_html( $learning_pathway_name ) . '</h1>
+				<!-- /wp:heading -->
+
+				<!-- wp:paragraph {"style":{"typography":{"lineHeight":1.6}}} -->
+				<p style="line-height:1.6">' . esc_html( $learning_pathway_description ) . '</p>
+				<!-- /wp:paragraph -->
+
+			</div>
+			<!-- /wp:group -->
+
+			<!-- wp:group {"style":{"layout":{"selfStretch":"fixed","flexSize":"33%"},"background":{"backgroundImage":{"url":"' . esc_url( get_stylesheet_directory_uri() . '/assets/learning-pathway-' . $learning_pathway_slug . '.svg' ) . '","source":"file"},"backgroundPosition":"0% 50%"}},"layout":{"type":"constrained"}} -->
+			<div class="wp-block-group"></div>
+			<!-- /wp:group -->
+
+		</div>
+		<!-- /wp:group -->
+
+	</div>
+	<!-- /wp:group -->
+	
+	<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"left":"var:preset|spacing|edge-space","right":"var:preset|spacing|edge-space"}}},"layout":{"type":"constrained"}} -->
+	<div class="wp-block-group alignfull" style="padding-right:var(--wp--preset--spacing--edge-space);padding-left:var(--wp--preset--spacing--edge-space)">
+
+		<!-- wp:group {"align":"wide","layout":{"type":"flex","flexWrap":"wrap","justifyContent":"space-between"},"style":{"spacing":{"margin":{"top":"var:preset|spacing|40","bottom":"var:preset|spacing|40"}}}} -->
+		<div class="wp-block-group alignwide" style="margin-top:var(--wp--preset--spacing--40);margin-bottom:var(--wp--preset--spacing--40)">
+
+			<!-- wp:search {"label":"' . __( 'Search', 'wporg-learn' ) . '","showLabel":false,"placeholder":"' . __( 'Search', 'wporg-learn' ) . '","width":290,"widthUnit":"px","buttonText":"' . __( 'Search', 'wporg-learn' ) . '","buttonPosition":"button-inside","buttonUseIcon":true,"query":{"wporg_learning_pathway":"' . esc_attr( $learning_pathway_slug ) . '"}} /-->
+
+			<!-- wp:group {"style":{"spacing":{"blockGap":"0"}},"layout":{"type":"flex","flexWrap":"nowrap"},"className":"wporg-query-filters"} -->
+			<div class="wp-block-group wporg-query-filters">
+				<!-- wp:wporg/query-filter {"key":"language"} /-->
+				<!-- wp:wporg/query-filter {"key":"topic"} /-->
+				<!-- wp:wporg/query-filter {"key":"level","multiple":false} /-->
+			</div>
+			<!-- /wp:group -->
+
+		</div>
+		<!-- /wp:group -->
+		
+	</div>
+	<!-- /wp:group -->';
+
+	$wrapper_attributes = get_block_wrapper_attributes();
+	$align = $attributes['align'];
+	if ( $align ) {
+		$wrapper_attributes = str_replace( 'class="', 'class="align' . $align . ' ', $wrapper_attributes );
+	}
```

Comment:
For some reason the `align` setting on the block markup isn't working, even though I've set the supports flag.